phpBB/styles/prosilver/template/mcp_post.html

```diff
@@ -168,6 +168,10 @@
 			</form>
 		<!-- ENDIF -->
 
+		<!-- IF S_MCP_POST_ADDITIONAL_OPTS -->
```

Comment:
Well just imagine an extension wants to display something with:
 - S_CAN_LOCK_POST = true
- S_MCP_POST_ADDITIONAL_OPTS = false

It's not possible because you wrapped the event. So just remove that IF around the event.